irony-completion.el

```diff
@@ -259,9 +276,13 @@ A candidate is composed of the following elements:
  6. Post-completion data. The text to insert followed by 0 or
     more indices. These indices work by pairs and describe ranges
     of placeholder text.
-    Example: (\"(int a, int b)\" 1 6 8 13)"
-  (and (irony-completion-candidates-available-p)
-       irony-completion--candidates))
+    Example: (\"(int a, int b)\" 1 6 8 13)
+ 7. The availability of the candidate."
+  (when (irony-completion-candidates-available-p)
+    (cl-loop for candidate in irony-completion--candidates
+             when (member (irony-completion-availability candidate)
```

Comment:
Replace `member` by `memq` since we are dealing with symbols.

The `cl-loop` is doing a filter operation, so I would use the more descriptive `cl-remove-if-not`.